craft_parts/plugins/maven_plugin.py

```diff
@@ -38,6 +38,7 @@ class MavenPluginProperties(PluginProperties, frozen=True):
     plugin: Literal["maven"] = "maven"
 
     maven_parameters: list[str] = []
+    maven_use_mvnw: bool = False
```

Comment:
maybe this variable should be `maven_use_wrapper` instead? Feels nicer to look at, and then gradle's can be `gradle_use_wrapper`

Reply:
This is reads a lot better, changed, thank you :)